tool/src/subprocess_launcher.dart

```diff
@@ -174,3 +176,21 @@ class SubprocessLauncher {
 
   static final _quotables = RegExp(r'[ "\r\n\$]');
 }
+
```

Comment:
__considered__ extending https://api.dart.dev/stable/3.3.0/dart-io/ProcessException-class.html ?

Reply:
For all of those folks who want to catch...uh...

You know, not a biggy. Just pondering...